### PR TITLE
add readme file

### PR DESCRIPTION
**Pull request by ftjahjad**
Add readme for loop device and nfs connectathon.

### CODE REVIEW

File: filesystems/loopdev/sanity/README.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# loopdevie sanity test suite
+Sanity test suite for loop device. This test  will mount various filesystems, and stress them through iozone. \
```
Comment (emanchado) [inline filesystems/loopdev/sanity/README.md]: I can barely believe I'm even mentioning this, but there are two spaces between "test" and "will" 😅

File: filesystems/loopdev/sanity/README.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+Test Maintainer: [Xiong Zhou](mailto:xzhou@redhat.com)
+
+## How to run it
+
+### Dependencies
+Please refer to the top-level README.md for common dependencies. Test-specific dependencies will automatically be installed when executing 'make run'. For a complete detail, see PURPOSE file
```
Comment (emanchado) [inline filesystems/loopdev/sanity/README.md]: Missing full stop at the end of the last sentence 😅

Comment (ftjahjad) [reply filesystems/loopdev/sanity/README.md]: samething. :sweat_smile:

File: filesystems/loopdev/sanity/README.md (hunk 1 above)
Comment (ftjahjad) [reply filesystems/loopdev/sanity/README.md]: ha, Thanks, not realize it adding two spaces. 
Thanks @emanchado  :+1:

**Pull request merged**